ge/org.osate.ge.fx/src/org/osate/ge/fx/palette/PaletteItem.java

```diff
@@ -0,0 +1,79 @@
+package org.osate.ge.fx.palette;
+
+import java.util.Objects;
+
+import javafx.beans.value.ObservableValue;
+import javafx.beans.value.WeakChangeListener;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Region;
+import javafx.scene.text.Font;
+
+class PaletteItem<I> extends Region {
+
+	private final Button Button;
+	private boolean Hovered;
+	private final PaletteModel<?, ?> model;
+	private final I item;
+	private static final String HOVER_ITEM_STYLE = "-fx-background-color: rgba(252,228,179,1.0);";
```

Comment:
Please move static constants to top of class for consistency with the project as a whole.

Reply:
I have made the above changes.